scanner/src/main/kotlin/scanners/ScanCode.kt

```diff
@@ -53,7 +54,8 @@ object ScanCode : LocalScanner() {
     override val resultFileExt = "json"
 
     override fun bootstrap(): File? {
-        val scancodeDir = File("src/funTest/assets/scanners/scancode-toolkit")
+        val gitRoot = File(".").searchUpwardsForSubdirectory(".git")
+        val scancodeDir = File(gitRoot, "scanner/src/funTest/assets/scanners/scancode-toolkit")
```

Comment:
How about one more sanity check after this line:

    if (!scancodeDir.isDirectory) return null

This way we return early if `scancodeDir` (or `gitRoot`) is `null`, e.g. because ORT was run from a distribution and not from a Git checkout.

Reply:
Wouldn't a distribution have `scancode` in the `PATH` and thus not use this code path at all?

Reply:
But sure, I can add that. Do you want it as a separate commit, or added to the one commit already there?

Reply:
Well, we cannot rely on that. Our distribution just comes with ORT, and it depends on the user what he / she has installed on his / her system.

Reply:
Please just amend it to your existing commit.

Reply:
Amended.